packages/maskbook/package.json

```diff
@@ -75,6 +75,7 @@
     "date-fns": "^2.23.0",
     "elliptic": "^6.5.3",
     "ethereumjs-util": "^7.1.0",
+    "ethers": "^5.4.4",
```

Comment:
It's not being used?

Reply:
It is used on `utils/bmath.ts`

Reply:
I tried before adding this library but it is not possible, they have different logic and they are not interchangeable easily.
ethers bignumber only works on integers but bignumber.js uses floating-point arithmetic.